src/renuka/BankingSystem.java

```diff
@@ -0,0 +1,73 @@
+package renuka;
+
+public class BankingSystem {
+	
+	String custName;
+	int totalAmt;
+	int creditCount = 0;
+	int debitCount = 0;
+	int balanceCount = 0;
+	static int totalDebitCount = 0;
+	static int totalCreditCount = 0;
+	static int totalPrintBalanceCount = 0;
+	
+	
+	void customerDetails(String name, int totalAmount) {
+		custName = name;
+		totalAmt = totalAmount;
+	}
+	
+	void debitAmount(int dbtAmount) {
+		if(totalDebitCount >= debitCount)
```

Comment:
you should do bracket for entire code, think on this if someone give input which is not fulfill the requirement then still it will increase cnt for debit